src/scripts.js

```diff
@@ -31,8 +30,20 @@ const availableRooms = document.getElementById('availableRooms')
 const availableRoomsHeader = document.getElementById('availableRoomsHeader')
 const bookRoomSuccessPopup = document.getElementById('bookRoomSuccess')
 const noDateAvailablePopup = document.getElementById('noDatesAvailable')
+
+const networkErrorPopup =document.getElementById('networkError')
+const overlayMain = document.querySelector('.overlay-main')
+const overlayLogin = document.querySelector('.overlay-login')
+const userLoginForm = document.getElementById('userLoginForm')
+const usernameInput = document.getElementById('userInputID');
+const passwordInput = document.getElementById('userInputPassword');
+const userLoginButton = document.getElementById('userLoginButton')
+const unmatchedCredentialsError = document.querySelector('.unmatched-credentials-error')
+const emptyFieldsError = document.querySelector('.empty-fields-error')
+const loadingCircle = document.getElementById('loadingCircle')
+const welcomeMessage = document.getElementById('welcomeMessage')
 const networkErrorPopup = document.getElementById('networkError')
```

Comment:
This is already declared on `line 34` so it's throwing error. Once deleted, this should not be a problem!